manager/src/main/java/com/redhat/service/bridge/manager/connectors/ConnectorsService.java

```diff
@@ -9,11 +10,9 @@
 
 public interface ConnectorsService {
 
-    Optional<ConnectorEntity> createConnectorIfNeeded(BaseAction resolvedAction,
+    Optional<ConnectorEntity> persistConnectorIfNeeded(BaseAction resolvedAction,
```

Comment:
@lucamolteni Can we call this `createConnector` to align us with our other Service interfaces. The fact that it is returning an `Optional` to me means we don't need the `IfNeeded` suffix.